src/components/localization/localized-link.js

```diff
@@ -149,6 +151,8 @@ export const LocalizedLink = React.forwardRef(({ to, ...props }, ref) => {
             lang_to = getDerivAppLocalizedURL(deriv_bot_app_url, locale)
         } else if (is_mt5_link) {
             lang_to = getDerivAppLocalizedURL(`${deriv_app_url}/mt5`, locale)
+        } else if (is_derivx_link) {
+            lang_to = getDerivAppLocalizedURL(deriv_x_app_url, locale)
```

Comment:
You don't need to set this if you're not calling it on `LocalizedLink`.

Reply:
Noted, I'll resolve that